### DISCUSSION TITLE
.Reality Behaviour support

### DISCUSSION THREAD

**Discussion by ti-hardikshah** [google/model-viewer#2867] [Q&A]
### Description

Hello , I want to know .reality file supported in model-viewer ios-src tag ? 
i guess yes. 

But i am facing issue , i am using USDZ model only that play animation but if i am trying same with .reality with button tap behavior then its not working.

Animation event is working completely fine in  Reality Composer but when export and run on browser no animation at all. 

#### Live Demo
https://ti-hardikshah.github.io/UsExpo/index.html

### Browser Affected

- [ ] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [x] Safari

### OS

- [ ] Android
- [X] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR
- [ ] WebXR
- [ ] SceneViewer
- [X] QuickLook

**Comment by elalish** [comment]
All we do with .reality and  USDZ files is pass the `ios-src` URL to Quick Look. Anything  after that is an Apple bug; please file it on them.

**Reply by ti-hardikshah** [reply]
Ohh thanks for the quick replay. I'll try to contact with Apple for this. whatever i'll get i'll post it here so every one can get answer.

**Reply by elalish** [reply]
@grorg Here's another issue for you at Apple.
